packages/astro/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@vue-macros/astro",
+  "version": "0.0.0",
+  "packageManager": "pnpm@8.6.12",
+  "description": "Astro integration of Vue Macros.",
+  "keywords": [
+    "vue-macros",
+    "macros",
+    "vue",
+    "sfc",
+    "setup",
+    "script-setup",
+    "astro",
+    "astro-integration"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/vue-macros/vue-macros#readme",
+  "bugs": {
+    "url": "https://github.com/vue-macros/vue-macros/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/vue-macros/vue-macros.git",
+    "directory": "packages/astro"
+  },
+  "author": "三咲智子 <sxzz@sxzz.moe>",
```

Comment:
let's put your name on it, and change it to `contributors`, like
https://github.com/vue-macros/vue-macros/blob/d888406bafa52cd427c60900fbc0cac51c5e4ce7/packages/jsx-directive/package.json#L26-L30